examples/cfd/external_aerodynamics/domino/src/test.py

```diff
@@ -472,9 +476,7 @@ def main(cfg: DictConfig):
             surface_coordinates = np.array(mesh.cell_centers().points, dtype=np.float32)
 
             interp_func = KDTree(surface_coordinates)
```

Comment:
I haven't looked at this code.  But we might replace this with a cuml call to find the neighbors on the GPU.